sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/storage/blob/AzureStorageBlobAutoConfigurationTests.java

```diff
@@ -151,6 +191,31 @@ void configurationPropertiesShouldBind() {
             });
     }
 
+    @Test
+    void configurationPropertiesShouldBindWithGlobalConfig() {
+        String accountName = "test-account-name";
+        String connectionString = String.format(STORAGE_CONNECTION_STRING_PATTERN, accountName, "test-key");
```

Comment:
Shouldn't we have a test case with storage service properties set on the storage service level and storage blob/file/queue level, and the properties got set correctly?